### PR TITLE
Bug 1177417 - Fix identity popup ui class for sub panels

### PR DESCRIPTION
**Pull request by whimboo**

### CODE REVIEW

File: firefox_puppeteer/tests/test_toolbars.py [hunk 1]
```diff
@@ -2,8 +2,6 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-import unittest
```
Comment (AutomatedTester) [inline firefox_puppeteer/tests/test_toolbars.py]: Is unittest no longer needed. In other patches you have added it back

**Comment by AutomatedTester** [comment]
r+ with an answer to my question

File: firefox_puppeteer/tests/test_toolbars.py (hunk 1 above)
Comment (whimboo) [inline firefox_puppeteer/tests/test_toolbars.py]: It was only used for the @unittest.skip() decorator. Given that it has been removed we don't need it anymore in this file.

**Comment by whimboo** [comment]
Merged as https://github.com/mozilla/firefox-ui-tests/commit/07277539b3ed0ad4f1db2a1a1d25b5c55d40086a.

**Pull request closed without merging**